### PR TITLE
Feature/775 add new params to conv scripts

### LINKED ISSUE #775: Add experimental-mapping-rule & catalyst-workaround to convenience scripts
## Background
Dataset with 120 conformance rules. 20 records.
without experimental mapping rule and catalyst workaround 460.14s
with experimental mapping rule and catalyst workaround 188.40s 

This is a huge improvement and should IMHO be available through the scripts. Maybe even default set to true.

## Feature
Add experimental-mapping-rule & catalyst-workaround to convenience scripts

### PR DESCRIPTION
**Pull request by yruslan**
Fixes #775 and #806

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,7 +208,10 @@ The list of additional options available for running Standardization:
 | --csv-quote **character**  | Specifies a character to be used as a quote for creating fields that might contain delimiter character. By default `"` is used. |
 | --csv-escape **character** | Specifies a character to be used for escaping other characters. By default '&#92;' (backslash) is used. |
 | --trimValues **true/false** | Indicates if string fields of fixed with text data should be trimmed.        |
+| --is-xcom **true/false**   | If `true` a mainframe input file is expected to have XCOM RDW headers.        |
 | --folder-prefix **prefix** | Adds a folder prefix before the date tokens.                                  |
+| --experimental-mapping-rule **true/false** | If `true`, an experimental optimized mapping rule implementation is used. The default value is environment-specific and is set in configuration files. |
```
Comment (Zejnilovic) [inline README.md]: Which configuration files are you talking about?

Comment (yruslan) [reply README.md]: application.properties in Conformance:
```
# Use experimental mapping rule implementation that groups explosions for rules
# operating on the same array
conformance.mapping.rule.experimental.implementation=false

# Enable workaround for Catalyst execution plan optimization freeze
conformance.catalyst.workaround=true
```

Comment (Zejnilovic) [reply README.md]: Then it is a build specific, not environment specific. I view these 2 as quite different. One is easy to change, the second one needs a full new build. For this, we have #510 for extraction of properties for Menas. Maybe create something even for standardization and conformance?

Comment (yruslan) [reply README.md]: Ok, will change to 'build specific'.

**Pull request merged**